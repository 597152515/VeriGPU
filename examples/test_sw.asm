# test sw

li x2, 0xf0
li x1, 123
outr x1
outr x2
sw x1, 0(x2)
outloc 0xf0

li x2, 0xf4
li x1, 111
sw x1, 0(x2)
outloc 0xf4

outloc 0xf0
outloc 0xf4

halt
