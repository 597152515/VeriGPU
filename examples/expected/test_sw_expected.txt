out 0 00000000000000000000000001111011 0000007b 123
out 1 00000000000000000000000011110000 000000f0 240
out 2 00000000000000000000000001111011 0000007b 123
out 3 00000000000000000000000001101111 0000006f 111
out 4 00000000000000000000000001111011 0000007b 123
out 5 00000000000000000000000001101111 0000006f 111
